### DISCUSSION TITLE
Diarization not working as intended

### DISCUSSION THREAD

**Discussion by naman-statisfy** [deepgram/community#470] [General help]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

I am trying to transcribe an audio file using with 3 different speakers in it. I am using python sdk as mentioned in the website tutorial. I am passing 'diarize' as True. However I am getting 'speaker' field as 0 for all the speakers. Is there anything else that I am missing? 
I have attached the audio file if it helps. Thanks

### If possible, please attach your code or paste it into the text box.

    from deepgram import Deepgram
    import asyncio, json, sys

    DEEPGRAM_API_KEY = '<>'

    FILE = './firstMinute.mp3'

    MIMETYPE = 'audio/wav'

    async def main():

      deepgram = Deepgram(DEEPGRAM_API_KEY)

      if FILE.startswith('http'):
        source = {
          'url': FILE
        }
      else:
        audio = open(FILE, 'rb')

        source = {
          'buffer': audio,
          'mimetype': MIMETYPE
        }

      response = await asyncio.create_task(
        deepgram.transcription.prerecorded(
          source,
          {
            'smart_format': True,
            'model': 'nova-2',
            'diarize': True
          }
        )
      )

      print(response)

    try:
      asyncio.run(main())
    except Exception as e:
      exception_type, exception_object, exception_traceback = sys.exc_info()
      line_number = exception_traceback.tb_lineno
      print(f'line {line_number}: {exception_type} - {e}')

### If possible, please attach an example audio file to reproduce the issue.

[firstMinute.zip](https://github.com/deepgram/community/files/13515029/firstMinute.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
